src/ensembl/src/content/app/browser/drawer/Drawer.scss

```diff
@@ -14,30 +16,38 @@
     text-align: center;
   }
 
-  dd {
-    width: 600px;
+  p {
+    margin: 0px;
   }
 
-  label {
-    float: left;
-    font-weight: $light;
-    text-align: right;
-    width: 150px;
+  .nextLine {
+    display: block;
+    margin-bottom: 10px;
   }
 
-  p {
-    margin: 0 0 20px 0;
+  .clearFix {
+    display: grid;
+    grid-template-columns: 100px 1fr;
+    grid-column-gap: 1em;
+    grid-row-gap: 1em;
+  }
+
+  .label {
+    font-weight: $light;
+    text-align: right;
+    width: 90px;
   }
 
   .details {
-    float: right;
+    display: flex;
     text-align: left;
-    width: 400px;
+    width: 490px;
```

Comment:
Why the 490px?

If we want the details field to be smaller, let's do this via the grid itself:

```
something-that-has-display-grid;
max-width: 600px;
```

or something to that effect, if that is indeed what we want.

Reply:
Remove width completely, this was from previous commits when using float :)